app/src/main/java/fr/free/nrw/commons/WelcomeActivity.java

```diff
@@ -50,10 +48,17 @@ public void onCreate(Bundle savedInstanceState) {
             findViewById(R.id.finishTutorialButton).setVisibility(View.VISIBLE);
         }
 
-        ButterKnife.bind(this);
+        binding = ActivityWelcomeBinding.inflate(getLayoutInflater());
+        final View view = binding.getRoot();
+        setContentView(view);
+
+        pager = binding.welcomePager;
+        indicator = binding.welcomePagerIndicator;
 
         pager.setAdapter(adapter);
         indicator.setViewPager(pager);
+
+        pager.setOnClickListener(this::finishTutorial);
```

Comment:
Comparing with master, I believe your click listener is being bound to the wrong component here.  Originally Butterknife was binding `R.id.finishTutorialButton`, so this should be `binding.finishTutorialButton`.

Instead of using a reference, you could use a lambda here -
```
binding.finishTutorialButton.setOnClickListener(v -> finishTutorial());
```
and you wont need to add the "view" parameter, which should also let you roll-back the change in the view pager call.